src/upchunk.spec.ts

```diff
@@ -311,11 +311,15 @@ test('abort pauses the upload and cancels the current XHR request', (done) => {
   const chunkSuccessCallback = jest.fn();
 
   upload.on('attempt', (e) => {
+    /*
+     *  - This timeout of 100ms is arbitrary. If this timeout is too low we have a race condition because the nocked 'scope' needs to be done
+     *  - The failure case of the race condition is that scope.isDone() can potentially be false
+     */
     setTimeout(() => {
       expect(scope.isDone()).toBeTruthy();
       expect(chunkSuccessCallback).toHaveBeenCalledTimes(0);
       done();
-    }, 10);
+    }, 100);
```

Comment:
There seems to be a race condition in the test. Described by the comment I added there. With the 10ms timeout it was failing intermittently in GH actions.

Is this a 🩹 ? Yes
Did I fix it so that the test passes in GH actions? Yes
Is there still a race condition in this test? Yes

Now that I'm thinking about it, do we actually _need_ that check for `expect(scope.isDone()).toBeTruthy()`?